pom.xml

```diff
@@ -210,7 +210,7 @@
     <maven.site.plugin.version>3.12.1</maven.site.plugin.version>
     <maven.spotbugs.plugin.version>4.7.2.1</maven.spotbugs.plugin.version>
     <maven.pmd.plugin.version>3.20.0</maven.pmd.plugin.version>
-    <pmd.version>6.50.0</pmd.version>
+    <pmd.version>6.54.0</pmd.version>
```

Comment:
https://github.com/pmd/pmd/releases
6.54 was released on 1/28.
6.51 was released on 10/29.
6.50 was released on 9/30.

Reply:
https://app.circleci.com/pipelines/github/checkstyle/checkstyle/17201/workflows/932a838c-765b-4619-85fb-64b47b8927f8/jobs/240411?invite=true#step-103-96
> You have 26 PMD violations.

Reply:
> [WARNING] Unable to exclude rules [BeanMembersShouldSerialize] from ruleset reference category/java/errorprone.xml; perhaps the rule name is misspelled or the rule doesn't exist anymore?
[WARNING] Discontinue using Rule name category/java/design.xml/ExcessiveMethodLength as it is scheduled for removal from PMD. PMD 7.0.0 will remove support for this Rule.

6.52
https://github.com/pmd/pmd/releases/tag/pmd_releases%2F6.52.0
> The Java rule [BeanMembersShouldSerialize](https://pmd.github.io/pmd-6.52.0/pmd_rules_java_errorprone.html#beanmembersshouldserialize) has been renamed to
[NonSerializableClass](https://pmd.github.io/pmd-6.52.0/pmd_rules_java_errorprone.html#nonserializableclass).

Reply:
https://pmd.github.io/latest/pmd_rules_java_design.html#immutablefield
Property ignoredAnnotations is listed as Deprecated.